packages/server/src/migration/migs/0.2.0-user-token.ts

```diff
@@ -0,0 +1,29 @@
+import { Migration } from '../migration'
+
+export class UserTokenMigration extends Migration {
+  meta = {
+    name: UserTokenMigration.name,
+    description: 'Adds token column to the msg_users table',
+    version: '0.2.0'
+  }
+
```

Comment:
The file name is not important for this migration service. All needed metadata is defined here. The migrations are executed in order using the version property and alphabetically using the name property. So normally if we give the migration source file the same name as the class, and prefix it with the migration version, the migs folder should display the migrations in the order that they are executed.

Reply:
Most migration services or libraries I have worked with uses a timestamp as a prefix so you don't have to use anything else for the order.

In our case, correct me if I'm wrong, but since we are using a version number + class name for the order, it means that two migrations that point to the same version might have to choose a name that allows them to have the proper order of execution?

E.g. I have 0.2.0 - UserToken and 0.2.0 - UserVersion (version - class). But, if for some reason, the latter needs to be executed first, I must change its name to something that alphabetically comes before 'UserToken'.

Reply:
Yes in that case you would need to prefix the migration with a number. For example you could name the files `0.2.0-1-user-version.ts` and `0.2.0-2-user-token.ts` and in the name property you do `'1' + UserVersionMigration.name` and `'2' + UserTokenMigration.name`.

Normally I'd expect migrations in the same version to not be dependant on each other. Otherwise they could just be the same migration. So overall this is a minor problem. It would annoy me more to have random numbers at the start of every migration file name. We don't use migration template generators so we'd need to put these timestamps here ourselves.